packages/web/src/common/store/notifications/fetchNotifications.ts

```diff
@@ -1,8 +1,12 @@
-import { IntKeys, FeatureFlags } from '@audius/common/services'
-import { getContext } from '@audius/common/store'
+import {
```

Comment:
this is awesome. such much less work here.
im not 100% sure, though probably 95% sure that it's ok to drop the identity stuff, but can we run a some quick numbers to just check what the impact is?

Reply:
For posterity:
```
-- Discovery users without notifs
SELECT COUNT(*) AS users_without_notifs
FROM users
WHERE NOT EXISTS (
	SELECT 1
	FROM notification n
	WHERE ARRAY[users.user_id] && n.user_ids
);

= 2046836

-- Identity users without notifs
SELECT COUNT(*) AS users_without_notifs
FROM "Users" u
WHERE NOT EXISTS (
	SELECT 1
	FROM "Notifications" n
	WHERE n."userId" = u."id"
) AND NOT EXISTS (
	SELECT 1
	FROM "SolanaNotifications" s
	WHERE s."userId" = u."id"
);
= 2394813

-- Identity users without notifs after the fallback date
SELECT COUNT(*) AS users_without_notifs
FROM "Users" u
WHERE NOT EXISTS (
	SELECT 1
	FROM "Notifications" n
	WHERE n."userId" = u."id"
		AND n."createdAt" > '04-01-2023'
) AND NOT EXISTS (
	SELECT 1
	FROM "SolanaNotifications" s
	WHERE s."userId" = u."id"
		AND s."createdAt" > '04-01-2023'
);
= 2881566
```

So roughly 486,753 users haven't gotten a notif since 04-01-2023 and would go from having a notification in their notification drawer to not having one.

Reply:
Well that conclusion might be off, since the cutoff doesn't seem too hardlined on DN.. So probably fine